eng/Build.ps1

```diff
@@ -160,10 +162,10 @@ function BuildSolution() {
         /p:Publish=$publish `
         /p:ContinuousIntegrationBuild=$ci `
         /p:OfficialBuildId=$officialBuildId `
-        /p:BootstrapBuildPath=$bootstrapDir `
         /p:QuietRestore=$quietRestore `
         /p:QuietRestoreBinaryLog=$binaryLog `
         /p:TestTargetFrameworks=$testTargetFrameworks `
+        /v:$verbosity `
```

Comment:
Since we produce binlogs I'd like to not see this; helps keep the output clean.

Reply:
Hmmm....but how do I debug the proto build process without this?  My guess is most people don't know what a `binlog` is nor how to look at it....at least I don't... :)

If I do `build -v:n -bootstrap` from the command line I kind of expect to see a `/v:n` output for the bootstrap build....